### ISSUE TITLE
[ERR_UNSUPPORTED_DIR_IMPORT]: Directory import in GqlManager.ts

### ISSUE THREAD

**Issue by adamazad** [ensdomains/ensjs-v3#140]
I'm using the library in Node v19. The dynamic `import(graphql/language)` in GqlManager.ts is causing runtime errors

https://github.com/ensdomains/ensjs-v3/blob/dda951685ab822ff9fba318abbf51cf33d6b6214/packages/ensjs/src/GqlManager.ts#L107-L127

Causes the:

```
    Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/Users/adam/workspace/nimi/nimi-services/node_modules/graphql/language' is not supported resolving ES modules imported from /Users/adam/workspace/nimi/nimi-services/node_modules/@ensdomains/ensjs/dist/esm/GqlManager.mjs
Did you mean to import graphql/language/index.js?
    at new NodeError (node:internal/errors:393:5)
    at finalizeResolution (node:internal/modules/esm/resolve:253:17)
    at moduleResolve (node:internal/modules/esm/resolve:876:10)
    at defaultResolve (node:internal/modules/esm/resolve:1084:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:831:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:413:18)
    at ESMLoader.import (node:internal/modules/esm/loader:514:22)
    at importModuleDynamically (node:internal/modules/esm/translators:110:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:36:14)
```

**Comment by adamazad** [comment]
Moving the dynamic import up in the file won't help. After affect: 

```
Debug: handler, error 
    Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/adam/workspace/nimi/nimi-services/node_modules/@ensdomains/ensjs/dist/esm/functions/initialGetters' imported from /Users/adam/workspace/nimi/nimi-services/node_modules/@ensdomains/ensjs/dist/esm/index.mjs
    at new NodeError (node:internal/errors:393:5)
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:876:10)
    at defaultResolve (node:internal/modules/esm/resolve:1084:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:831:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:413:18)
    at ESMLoader.import (node:internal/modules/esm/loader:514:22)
    at importModuleDynamically (node:internal/modules/esm/translators:110:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:36:14)
Debug: internal, error 
    Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/adam/workspace/nimi/nimi-services/node_modules/@ensdomains/ensjs/dist/esm/functions/initialGetters' imported from /Users/adam/workspace/nimi/nimi-services/node_modules/@ensdomains/ensjs/dist/esm/index.mjs
    at new NodeError (node:internal/errors:393:5)
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:876:10)
    at defaultResolve (node:internal/modules/esm/resolve:1084:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:831:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:413:18)
    at ESMLoader.import (node:internal/modules/esm/loader:514:22)
    at importModuleDynamically (node:internal/modules/esm/translators:110:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:36:14)

```

**Comment by thomas779** [comment]
Bumping this as I encounter this issue too

**Comment by TateB** [comment]
fixed in v3.0.0 latest! `graphql/language` is no longer imported dynamically.
